src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -754,6 +754,39 @@ public CompletableFuture<GetBindingSuggestionsResponse> getBindingSuggestion(Get
     return backendServiceFacade.getBackendService().getBindingSuggestion(params);
   }
 
+  @Override
+  public CompletableFuture<Void> changeIssueStatus(ChangeIssueStatusParams params) {
+    var coreParams = new org.sonarsource.sonarlint.core.clientapi.backend.issue.ChangeIssueStatusParams(
+      params.getConfigurationScopeId(), params.getIssueKey(), params.getNewStatus(), params.isTaintIssue());
+    return backendServiceFacade.getBackendService().changeIssueStatus(coreParams).thenAccept(nothing -> {
+      var key = params.getIssueKey();
+      if (params.isTaintIssue()) {
+        taintVulnerabilitiesCache.removeTaintIssue(params.getFileUri(), key);
+      } else {
+        issuesCache.removeIssueWithServerKey(params.getFileUri(), key);
+      }
+
+      diagnosticPublisher.publishDiagnostics(create(params.getFileUri()), false);
+      client.showMessage(new MessageParams(MessageType.Info, "Issue status was successfully changed"));
+    }).exceptionally(t -> {
+      lsLogOutput.error("Error changing issue status", t);
+      client.showMessage(new MessageParams(MessageType.Error, "Could not change status for the issue. Look at the SonarLint output for details."));
+      return null;
+    });
+  }
+
+  @Override
+  public CompletableFuture<Void> addIssueComment(AddIssueCommentParams params) {
+    return backendServiceFacade.getBackendService().addIssueComment(params)
+      .thenAccept(nothing -> client.showMessage(new MessageParams(MessageType.Info, "New comment was successfully added")))
```

Comment:
same as above - `new comment was added`

Reply:
Addressed in [a7b9dce](https://github.com/SonarSource/sonarlint-language-server/pull/246/commits/a7b9dce760338ff89c0fff30f0dad22aaed30606)